### ISSUE TITLE
`add.run` command adds code but `ls` can't view it

### ISSUE THREAD

**Issue by rlmark** [unisonweb/unison#3417]
Earlier today, we used the `add.run` command to add a term to the codebase. The UCM confirmed it was successfully added as a term. However, the term did not show up when running `ls` shortly thereafter. We were able to `display` the term, and the term was successfully pushed (and rendered) to a remote namespace, so it is definitely present in the codebase, but `ls` won't reveal its location. 

The version of the UCM which surfaced this is a very recent build, trunk was pulled Wed Sept 14, around 10 AM Pacific time, merged into the current LSP branch. (Git versioning was in a weird state) 

A transcript is attached to replicate this: 
[addRun.md](https://github.com/unisonweb/unison/files/9569970/addRun.md)

**Comment by aryairani** [comment]
@tstat Could you take a look at this?

**Comment by aryairani** [comment]
This seems to be fixed today:

````
```unison
main : '{IO,Exception} Text
main = do
  "hi"
```

```ucm

  Loading changes detected in scratch.u.

  I found and typechecked these definitions in scratch.u. If you
  do an `add` or `update`, here's how your codebase would
  change:
  
    ⍟ These new definitions are ok to `add`:
    
      main : '{IO, Exception} Text

```
```ucm
.> run main

  "hi"

.> add.run myFunction

  ⍟ I've added these definitions:
  
    myFunction : Text

.> ls

  1. builtin/   (643 terms, 92 types)
  2. myFunction (Text)

```
````
